modules/core/src/test/resources/int-value-expanded-with-links.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
This expanded output, even though being correct, is not convenient. Did you try with [java-jsonld](https://github.com/jsonld-java/jsonld-java) instead of jena? That library gives me the same output as Json-LD playground

Reply:
This library is too inconvenient to use. I give up on that for now.